### ISSUE TITLE
Items Randomly Getting Deleted In Inventory After Clicking On Them

### ISSUE THREAD

**Issue by DizzyDJW** [ybaspinar/Faruk-V#2]
So far, I've lost a granite machete after placing it from my sophisticated backpack into my hotbar, and while using it, I noticed it was no longer clearing nearby grass only breaking the grass I hit, then I opened my inv and the machete outright disappeared. The same happened TWICE, with a Crude Repair Kit full of Iron. I placed the repair kit into the crafting grid, and it got deleted instantly, then not long after I went to drag the new one I spawned back in, after having already successfully used it before, and it also disappeared.

**Comment by ybaspinar** [comment]
Which version of the modpack you are using? are you playing on server or single player?

**Comment by DizzyDJW** [comment]
Singleplayer
Modpack Version: 1.4
Minecraft Version: 1.16.5
Forge Version: 36.2.28

**Comment by ybaspinar** [comment]
Seems like its issue with Sophisticated Backpacks. Some mods change uuid of the backpack. I need bit more time to reproduce bug.

**Comment by ybaspinar** [comment]
In mean time you can use f3+h to check uuid of the backpack and compare it with /sbp list this command if theres difference you can click on the uuid to give it to you

**Comment by DizzyDJW** [comment]
UUIDs are exact match

**Comment by DizzyDJW** [comment]
Actually I have discovered the issue, it is Sophisticated Backpacks related. The crafting upgrade in the backpack was putting things into the crafting grid of the backpack even when the backpack was not open, and even when items were not shift-clicked. I replicated it twice. If I remove the upgrade the bug does not occur, but with the upgrade items, when interacted with in your inventory, in any way, have a chance to randomly disappear, without backpack UI open, and go into the crafting grid inside. No idea what is causing this though.
